cli/dcoscli/job/main.py

```diff
@@ -228,6 +233,39 @@ def _remove(job_id, stop_current_job_runs=False):
     return 0
 
 
+def _queue(job_id, json_flag=False, quiet=False):
+    """
+    :param job_id: Id of the job
+    :type job_id: str
+    :param json_flag: output json if True
+    :type json_flag: bool
+    :returns: process return code
+    :rtype: int
+    """
+    client = metronome.create_client()
+    deployment_list = client.get_queued_jobruns(job_id)
+
+    if not deployment_list and not json_flag:
+        msg = "There are no deployments in the queue"
+        if job_id:
```

Comment:
Why an exception is thrown if `job_id` evaluates to `True`?

Reply:
this is when the deployment_list doesn't exist... I considered an empty list to be normal however if you got an empty list and you specified a job_id then it is an error.   exit 0 is normal... errors are handled through exceptions in the CLI and will be transformed into a non-zero exit.    I believe this approach is consistent with previous marathon deployment code.